.github/workflows/stale-pull-requests.yml

```diff
@@ -0,0 +1,34 @@
+name: Check stale pull requests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 12 * * *'
+
+permissions:
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
+        with:
+          debug-only: true
+          # disable issues
+          days-before-issue-stale: -1
+          days-before-issue-close: -1
+          stale-pr-message: |
+            This pull request is stale because it has been open for 14 days with no activity.
+            It will be closed in 7 days unless the `stale` label is removed.
+          close-pr-message: |
+            This pull request was closed due to a lack of activity for 7 days after it was stale.
+          stale-pr-label: stale
+          close-pr-label: closed-due-inactivity
+          days-before-pr-stale: 14
```

Comment:
Now that I think about it, do you think it would make sense to use environment variables to define `days-before-pr-stale` and `days-before-pr-close`. E.g.
```
env:
  BEFORE_STALE: 14
  BEFORE_CLOSE: 7
```
And then define
```
days-before-pr-stale: $BEFORE_STALE
days-before-pr-close: $BEFORE_CLOSE
```
and also the messages could look like this
```
This pull request was closed due to a lack of activity for $BEFORE_CLOSE days after it was stale.
```
It would make it neater, with the added benefit that if we decide to change these values, we only have to do it in one place. Of course, I'm not sure if this is a proper approach :man_shrugging: 

Found the docs [here](https://docs.github.com/en/actions/learn-github-actions/environment-variables)

Reply:
Yes, definitely.